VAMobile/e2e/tests/utils.ts

```diff
@@ -458,21 +473,21 @@ export async function enableAF(AFFeature, AFUseCase, AFAppUpdate = false) {
       }
     }
   }
-  await element(by.id('AFTypeTestID')).replaceText(AFUseCase)
-  await element(by.id('AFTypeTestID')).tapReturnKey()
-  await element(by.id('AFErrorMsgTitleTestID')).replaceText('AF Heading Test')
-  await element(by.id('AFErrorMsgTitleTestID')).tapReturnKey()
-  await element(by.id('AFErrorMsgBodyTestID')).replaceText('AF Body Test')
-  await element(by.id('AFErrorMsgBodyTestID')).tapReturnKey()
+  await element(by.id(CommonE2eIdConstants.AF_TYPE_INPUT_ID)).replaceText(AFUseCase)
+  await element(by.id(CommonE2eIdConstants.AF_TYPE_INPUT_ID)).tapReturnKey()
+  await element(by.id(CommonE2eIdConstants.AF_ERROR_MSG_TITLE_INPUT_ID)).replaceText('AF Heading Test')
```

Comment:
I think you can change the value of 'AF Heading Test' to CommonE2eIdConstants.AF_ERROR_MSG_TITLE_ENTERED_TEXT and can probably do the same to 'AF Body Test'.